site/.vitepress/config.ts

```diff
@@ -474,11 +474,7 @@ export default defineConfig({
                 text: "constructor",
                 link: "/constructor",
               },
-              { text: "gasEstimator", link: "/gasEstimator" },
-              {
-                text: "withAlchemyGasManager",
-                link: "/withAlchemyGasManager",
-              },
```

Comment:
this should not be here, so removed. belongs to middleware, and it's already there down below.

Reply:
well, this is a field on the AlchemyProvider which calls the middleware method. i'd keep it. they are two separate docs

Reply:
@avasisht23 I put it back. Thanks :)